homework.md

```diff
@@ -44,43 +44,46 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used to make webpages interactive
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Allows multiple people to work collaboratively on the same project in remote branches without directly changing the main version of the project
+Allows to track history of the project and revert changes back if necessary by creating checkpoints in which versions of code are stored
+Allows to store a project remotely. Should anything happen to the hard disc the project can be retrieved from the remote repository
+
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You create a copy of that project on your account
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+You create a copy of that project on your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Sending changes you made in your cloned version of the project to the main project repository to be checked and (hopefully) merged with the main project
```

Comment:
That is correct, but, actually, you can create pull requests within the same repository as well. The usual flow in a company is that everyone has access to the repository, but instead of working on the main (sometimes master) branch, they work on their own branches. When they are ready, they create a PR to have their branch merged into main.

The fork-based flow is more common in open source projects.